### PR TITLE
🔥 Feature: recover middleware support print stack trace

### PR DESCRIPTION
**Pull request by tianjipeng**
**Please provide enough information so that others can review your pull request:**
The intention of this PR is to address the issue raised here: [1119](https://github.com/gofiber/fiber/issues/1119)
It allows for configuration of printing debug stacktrace.

**Explain the *details* for making this change. What existing problem does the pull request solve?**
Add variable to control whether print debug stacktrace  and can define custom print stack trace function.

**Commit formatting** 

Use emojis on commit messages so it provides an easy way of identifying the purpose or intention of a commit. Check out the emoji cheatsheet here: https://gitmoji.carloscuesta.me/

### CODE REVIEW

File: middleware/recover/config.go [hunk 1]
```diff
@@ -10,11 +10,25 @@ type Config struct {
...
 }
 
+var defaultStackTraceBufLen = 1024
+
 // ConfigDefault is the default config
 var ConfigDefault = Config{
-	Next: nil,
+	Next:             nil,
+	EnableStackTrace: false,
+	PrintHandler:     defaultPrintHandler,
```
Comment (kiyonlin) [inline middleware/recover/config.go:31]: I'm wondering if `StackTraceHandler` is more appropriate? It maybe does not just print the stack trace, but also can store it or send it to remote hosts.

**Review by kiyonlin — CHANGES_REQUESTED** [review]

File: middleware/recover/config.go (hunk 1 above)
Comment (tianjipeng) [reply middleware/recover/config.go]: Got it. ``StackTraceHandler`` is better than ``PrintHandler ``

**Review by kiyonlin — APPROVED** [review]
LGTM 😄

**Review by ReneWerner87 — APPROVED** [review]

**Comment by ReneWerner87** [comment]
Could you also add the new options in the gofiber docs repository
https://github.com/gofiber/docs/blob/master/api/middleware/recover.md#config

**Comment by tianjipeng** [comment]
@ReneWerner87 docs pr is here: https://github.com/gofiber/docs/pull/147

**Comment by Fenny** [comment]
LGTM

**Pull request merged**